docs/concepts/privacy.md

```diff
@@ -0,0 +1,73 @@
+
+So why doesn't the IPFS protocol itself explicitly have a "privacy layer" built in? This is in line with key principles of the protocol's highly modular design — after all, different uses of IPFS over its lifetime may call for different approaches to privacy. Explicitly implementing an approach to privacy within the IPFS core could "box in" future builders due to a lack of modularity, flexibility, and future-proofing. On the other hand, freeing those building on IPFS to use the best privacy approach for the situation at hand ensures IPFS is useful to as many as possible.
+
+If you're worried about the implications of this for your own personal use case, it's worth taking additional measures such as disabling reproviding, encrypting sensitive content, or even running a private IPFS network if that's appropriate for you. More details on these are below.
+
+::: tip
+While IPFS traffic _between nodes_ is encrypted, the essential metadata that nodes publish to the DHT — including their unique node identifiers (PeerIDs) and the CIDs of data that they're providing — is public. If you're worried about the implications of this for your personal use case, it's worth taking additional measures.
+:::
+
+## What's public on IPFS
+
+All traffic on IPFS is public, including the contents of files themselves (unless they're encrypted; more about this below). For purposes of understanding IPFS privacy, this may be easiest to think about in two halves: content identifiers (CIDs) and IPFS nodes themselves.
+
+### Content identifiers
+
+Because IPFS uses [content addressing](/concepts/content-addressing/) rather than the legacy web's method of location addressing, each piece of data stored in the IPFS network gets its own unique content identifier (CID). Copies of the data associated with that CID can be stored in any number of locations worldwide on any number of participating IPFS nodes. To make retrieving the data associated with a particular CID efficient and robust, IPFS uses a [distributed hash table](/concepts/dht/) (DHT) to keep track of what's stored where. When you use IPFS to retrieve a particular CID, your node queries the DHT to find the closest nodes to you with that item — and by default also agrees to reprovide that CID to other nodes for a limited time, until periodic "garbage collection" clears your cache of content you haven't used in a while. You can also "pin" CIDs that you want to make sure are never garbage-collected — either explicitly using IPFS's low-level `pin` API, or implicitly using the [Mutable File System](/concepts/file-systems/#mutable-file-system-mfs) (MFS) — which also means you're acting as a permanent reprovider of that data.
```

Comment:
Is the DHT lookup for all nodes or just DHT servers? Maybe also add the suggestion for how to reset your peerid

Reply:
(Sorry, this should be down on “identifiability” ex your peerid relating to your IP address)

Reply:
https://github.com/ipfs/ipfs-docs/pull/554/commits/b2e01c5cb2eebde792fb4cc74d2db84b7bc41a0a and https://github.com/ipfs/ipfs-docs/pull/554/commits/0afc26419f7d43fe1f4cd5b7968640df319eba42 add info on resetting PeerID.